src/pycmor/core/pipeline.py

```diff
@@ -260,6 +263,8 @@ class DefaultPipeline(FrozenPipeline):
         "pycmor.std_lib.units.handle_unit_conversion",
         "pycmor.std_lib.global_attributes.set_global_attributes",
         "pycmor.std_lib.variable_attributes.set_variable_attributes",
+        "pycmor.std_lib.dimension_mapping.map_dimensions",
+        "pycmor.std_lib.coordinate_attributes.set_coordinate_attributes",
```

Comment:
we should probably group these things into a submodule

Reply:
Maybe together with variable attributes?

Right now we have:
```

        "pycmor.std_lib.timeaverage.timeavg",
        "pycmor.std_lib.units.handle_unit_conversion",
        "pycmor.std_lib.global_attributes.set_global_attributes",
        "pycmor.std_lib.variable_attributes.set_variable_attributes",
        "pycmor.std_lib.dimension_mapping.map_dimensions",
        "pycmor.std_lib.coordinate_attributes.set_coordinate_attributes",
```

This could be simplified:

```suggestion
        "pycmor.std_lib.time.average",
        "pycmor.std_lib.units.convert",
        "pycmor.std_lib.attributes.set_global",
        "pycmor.std_lib.attributes.set_variable",
        "pycmor.std_lib.attributes.set_coordinates",
        "pycmor.std_lib.dimensions.map_dimensions",
```

That would however be a major renaming.